scripts/screener/screener.types.ts

```diff
@@ -118,4 +118,6 @@ export interface ScreenerProxyPayload {
   conclusion?: 'failure' | 'skipped' | 'cancelled';
   /** Name of the screener project for the screener run */
   project?: string;
+  /** Name of the branch */
+  branch?: string;
```

Comment:
```suggestion
  branch: string;
```

Let's  make this required so that it's harder to make the mistake again of not having it in the payload